src/contracts/BaseParaSwapAdapter.sol

```diff
@@ -80,8 +80,18 @@ abstract contract BaseParaSwapAdapter is IFlashLoanReceiverBase, Ownable {
    * @dev Get the vToken, sToken associated to the asset
    * @return address of the vToken
    * @return address of the sToken
+   * @return address of the aToken
    */
-  function _getReserveData(address asset) internal view virtual returns (address, address);
+  function _getReserveData(address asset) internal view virtual returns (address, address, address);
```

Comment:
I think it does look better with a struct but since we only use it internally and it's already correct, maybe would be increased gas for not much security/readability gain?